tty.go

```diff
@@ -71,7 +71,7 @@ func inheritStdio(process *libcontainer.Process) error {
 	return nil
 }
 
-func (t *tty) recvtty(process *libcontainer.Process, socket *os.File) error {
+func (t *tty) recvtty(process *libcontainer.Process, socket *os.File) (Err error) {
```

Comment:
First time I have ever seen `Err` instead of `err`

Reply:
Same here.  I kind of like it to differentiate and stop mistakes.

Reply:
So, this is quite widely used in a few places. The reason for not using `err` is because if you use a variable name already used in the function it can be a little bit confusing whether the last-assigned value of `err` is what's tested or the `return ___` which triggered the `defer`. Add in scopes like

```go
func baz() (err error) {
    if _, err := foo(); err == nil {
        defer func() {
            if err != nil {
               // does this ever execute?
            }
        }()
    }
}
```

And it just becomes complicated to reason about (even though the semantics are actually defined in the Go language spec -- in the above case [it does not execute](https://play.golang.org/p/wJyeNhe_vI5)).

By using a variable that is not used within the function at all, you eliminate these sorts of confusions and you also have a clear way of telling whether you are referencing the local variable or the actual return value. I use this quite a bit in `umoci` and I stole the idea from somewhere else (I think it's even used within Docker from memory).